### ISSUE TITLE
browser-bootstrap-from-source does not work

### ISSUE THREAD

**Issue by JensLincke** [LivelyKernel/lively-system-examples#1]
I had problems using the packaged version of lively-modules... with karma so I wanted to debug it and use the bootstrapped version. But this failed also:
## What I did:

Installed the repo

```
git clone https://github.com/LivelyKernel/lively-system-examples.git
npm install 
```

Served the directory....

```
http-server
```

Got the following error:

```
system.src.js:1059 GET http://localhost:8080/node_modules/lively.modules/node_modules/lively.vm/package.json 404 (Not Found)
```

**Comment by JensLincke** [comment]
The issue with karma btw was that lively.modules conflicted with the 

```
frameworks: ['requirejs'],
```

option in the karma.conf.js

**Comment by rksm** [comment]
That isn't really an issue with lively.modules but with npm, it puts the dependent packages in different places depending on the tree of dependencies. This is ugly but as long as we want to use npm for dependency installation unavoidable. With lively.next we will probably go a different route in the future. Anyway, I fixed the issue that made the example not run.

Apart from that, I would suggest following the browser or browser-bootstrap examples instead of browserbootstrap-from-source. The former ones are a lot simpler to implement. The latter is just there so we still remember how we got there ;)

I have no experience with karma or what a setup with it involves. For testing we use [mocha-es6](https://github.com/rksm/mocha-es6), our own wrapper around mocha that is aware of lively.modules. This covers all of our needs at this point (running tests in lively + in browser + in node.js).
